package.json

```diff
@@ -17,7 +17,7 @@
     "test:all:parallel": "npm run test:unit && npm run test:integration:parallel",
     "test:integration": "lerna run test:integration --stream --concurrency 1",
     "test:integration:parallel": "lerna run test:integration --stream --parallel",
-    "test:integration:windows": "lerna run test:integration --stream --ignore web-component-tester --ignore @polymer/esm-amd-loader --parallel",
+    "test:integration:windows": "lerna run test:integration --stream --ignore web-component-tester --ignore @polymer/esm-amd-loader --concurrency 1",
```

Comment:
is the concurrency 1 change intentional?

Reply:
I noticed some timeouts on appveyor when wct-mocha integration tests competed with wct integration tests on windows.  By eliminating parallelism on multiple selenium processes, this eliminated the randomness of the contention.  WCT's integration tests are flaky enough as it is, so this was intentional in the sense I was hoping to reduce the number of flaky runs on Windows which are made worse by introducing yet-another-selenium-integration-suite to the collection.